### ISSUE TITLE
[subset] assert if AdvWidthMap None, LsbMap & RsbMap non-None

### ISSUE THREAD

**Issue by blueshade7** [fonttools/fonttools#1587]
HVAR subsetter has the following line of assert in `subset/__init__.py` expecting if AdvWidthMap is missing, then LsbMap and RsbMap must be also both missing.
```
assert table.LsbMap is None and table.RsbMap is None, "File a bug."
```
OpenType spec does not appear to state such a requirement.
The attached test font triggering the assert passes both ots-sanitize and ttx.

VVAR subsetter also has a similar assert.
[TestHVVAR.ttf.zip](https://github.com/fonttools/fonttools/files/3109489/TestHVVAR.ttf.zip)

**Comment by behdad** [comment]
Right. Just not implemented.

**Comment by blueshade7** [comment]
Let me take care on this. My HVAR/VVAR fix for #1446 appears to fix this as well.
